pkg/topologyimpl/hivemetastore/topology.go

```diff
@@ -14,10 +14,11 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package hivemetastore
+package main
```

Comment:
after this pr, may refactor the package and put these plugin code in pkg/plugin/hivemetastore?

Reply:
yes, refactoring my next step, plugin code should be separated from topology implementation